appyters/Researcher_Summary_Report_Appyter/researcher_summary_appyter.ipynb

```diff
@@ -0,0 +1,746 @@
+   "source": [
+    "%%appyter code_exec\n",
+    "{% set researcher_name = StringField(\n",
+    "    name='researcher_name', \n",
+    "    label='Investigator Name', \n",
+    "    default=\"Robert J. Lefkowitz\", \n",
+    "    description='Input full name of PI along with middle names/middle initials followed by spaces as leaving it out may change the results of certain information', \n",
+    "    section='Data_Section1', \n",
+    "    required = True\n",
+    "    )\n",
+    "%}\n",
+    "{% set orcid = StringField(\n",
+    "    constraint= \"[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{4}|.{0}|[0-9]{4}\",\n",
+    "    name='researcher_id', \n",
+    "    label='(Optional) Investigator Orcid ID if Known', \n",
+    "    default='', \n",
+    "    description='Input the ORCID associated with the researcher if known in addition to the inputted name.', \n",
+    "    section='Data_Section1'\n",
+    "    )\n",
+    "%}\n",
+    "researcher_name = {{researcher_name}}\n",
+    "orcid_id = {{orcid}}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "split_name = researcher_name.split()\n",
+    "name_of_researcher = researcher_name.split()[0] + \" \" + researcher_name.split()[-1]\n",
+    "pubmed_name = split_name[-1] + \" \" +split_name[0][0]\n",
+    "if len(split_name) > 2:\n",
+    "    pubmed_name = split_name[-1] + \" \"\n",
+    "    for name in split_name[:-1]:\n",
+    "        pubmed_name += name[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "%%appyter markdown\n",
+    "# Researcher Summary Report for {{researcher_name.raw_value}}\n",
+    "This notebook takes as input a researcher name and outputs information about the researcher from various sources including PubMed, RePORTER, Google Scholar, Geneshot, and Drugshot. The information that is currently included in the report is publications, grants, citations, collaborations, associated genes and drugs, and additional information related to the researcher's affiliation and topics of research interests. The report provides graphs and tables. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#import packages\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import json\n",
+    "import os\n",
+    "#pd.options.display.max_rows = 50000\n",
+    "from time import sleep\n",
+    "from Bio import Entrez\n",
+    "from collections import defaultdict\n",
+    "import requests\n",
+    "import plotly.express as px\n",
+    "import plotly\n",
+    "from IPython.display import display,FileLink, HTML, Markdown, IFrame\n",
+    "from scholarly import scholarly\n",
+    "from wordcloud import WordCloud\n",
+    "import matplotlib.pyplot as plt\n",
+    "import datetime\n",
+    "import math\n",
+    "import re\n",
+    "import ipycytoscape\n",
+    "from py2neo import Graph\n",
```

Comment:
Please cleanup unused imports and remove unnecessary dependencies.

`py2neo` for example

Reply:
Cleaned up the imports